com.microsoft.mrtk.graphicstools.unity/Samples~/Experimental/Magnifier/Scenes/MagnifierShowcase.unity

```diff
@@ -0,0 +1,2054 @@
+%YAML 1.1
```

Comment:
For this scene could you add a FlyCamera component so users can move around in editor? 
![image](https://user-images.githubusercontent.com/13305729/199794756-4795f982-baa4-40cb-be38-ffc2f039e06b.png)